pkg/upload/react-native-ios.go

```diff
@@ -151,7 +157,13 @@ func ProcessReactNativeIos(
 					bundlePath = possibleBundleFilePath
 					log.Info("Found bundle file at: " + bundlePath)
 				} else {
-					log.Info("No bundle file found at: " + possibleBundleFilePath)
+					possibleBundleFilePath = filepath.Join(buildSettings.ProjectTempRoot, "ArchiveIntermediates", scheme, "BuildProductsPath", filepath.Base(buildSettings.BuiltProductsDir), "main.jsbundle")
+					if utils.FileExists(possibleBundleFilePath) {
+						bundlePath = possibleBundleFilePath
+						log.Info("Found bundle file at: " + bundlePath)
```

Comment:
Same for here, to state we couldn't find the bundle file in the expected location too.